cmd/minishift/cmd/addon/addons_list.go

```diff
@@ -63,13 +64,13 @@ func init() {
 	var err error
 	defaultListTemplate, err = template.New("list").Parse(defaultAddonListFormat)
 	if err != nil {
-		glog.Errorln("Error creating list template:", err)
+		fmt.Println("Error creating list template: ", err)
```

Comment:
@hferentschik you just stole my idea :), that something I had in mind, was going through the changes.